ArduPlane/mode_rtl.cpp

```diff
@@ -45,7 +45,7 @@ void ModeRTL::navigate()
 
     if (plane.g.rtl_autoland == 1 &&
         !plane.auto_state.checked_for_autoland &&
-        plane.reached_loiter_target() && 
+        plane.reached_loiter_target() &&
```

Comment:
this is unrelated white space

Reply:
Will fix, my editor automatically trims trailing whitespace.